website/content/en/docs/best-practices/observability-best-practices.md

```diff
@@ -0,0 +1,255 @@
+---
+title: "Operator Observability Best Practices"
+linkTitle: "Observability Best practices"
+weight: 6
+description: This guide describes the best practices concepts for adding Observability to operators.
+---
+
+## Operator Observability Best Practices
+
+In this document we will outline what operators require in order to meet the "Deep Insights" capability level and provide best practices and examples for creating metrics, recording rules and alerts. It is based on the general guidelines in [Operator Capability Levels](https://sdk.operatorframework.io/docs/overview/operator-capabilities/).
```

Comment:
What do you mean with `recording rules`?

Reply:
I guess this may refer to [prometheus recording rules](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/#recording-rules).
In general, the prometheus user would compose various `prom_ql` to calculate the status of the object under monitoring.
Well, some `prom_ql` maybe frequently applied and computed, others may be "computationally expensive".
A `recording rule` is like an alias to one of a `prom_ql` that is frequently used, so that prometheus will "record" the value of the query before you actually call it, so that save time.